client/fingerprint/cpu.go

```diff
@@ -80,6 +81,7 @@ func (f *CPUFingerprint) Fingerprint(req *FingerprintRequest, resp *FingerprintR
 			f.logger.Debug("detected reservable cores", "cpuset", reservableCores)
 		}
 	}
+	resp.AddAttribute("cpu.num_reservable_cores", strconv.Itoa(len(reservableCores)))
```

Comment:
the other `cpu.` attributes contain no underscores, 

```
cpu.num_reservable_cores  = 24
cpu.numcores              = 24
cpu.totalcompute          = 112128
```

what about naming this `cpu.reservablecores` ?

Reply:
Good call, thanks. Fixing.